JS-Algorithms/challenges.js

```diff
@@ -41,7 +52,12 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  const indexItem = index % COUNTRY_NAMES.length;
+  const firstPartOfArray = COUNTRY_NAMES.slice(indexItem);
+  const secondPartOfArray = COUNTRY_NAMES.slice(0, indexItem);
+  const newArray = [...firstPartOfArray, ...secondPartOfArray];
```

Comment:
you can return this directly right? 
```suggestion
  return [...firstPartOfArray, ...secondPartOfArray];
```

Reply:
also, why spread operator is needed?

Reply:
Because I can get a mixed single array, otherwise I would get nested arrays or something else.